pom.xml

```diff
@@ -1091,17 +1091,11 @@
 <!-- Polly end -->
 
 <!-- ProgramAB begin -->
-  <dependency>
-    <groupId>program-ab</groupId>
-    <artifactId>program-ab-data</artifactId>
-    <version>1.1</version>
-    <scope>provided</scope>
-    <type>zip</type>
-  </dependency>
+<!-- skipping program-ab program-ab-data program-ab-program-ab-data-null null version/latest -->

```

Comment:
Where are the default bots coming from if this isn't here?

Reply:
runtime ivy gets the latest program-ab-data - same as inmoov2
we'll pin it on release